index.html

```diff
@@ -31,21 +30,17 @@
             <div class="hamburger-icon">
                 <img src="images/icon/menu-icon.png" class="menu-icon" alt="menu icon">
             </div>
-            <div class="nav-list">
+            <div class="desktop">
                 <ul class="nav-items">
-                    <li>
-                        <a href="#" class="nav-item">Portfolio</a>
-                    </li>
-                    <li>
-                        <a href="#" class="nav-item">About</a>
-                    </li>
-                    <li>
-                        <a href="#" class="nav-item">Contact</a>
+                    <li><a href="#" class="nav-item">Portfolio</a></li>
+                    <li><a href="#" class="nav-item">About</a></li>
+                    <li><a href="#" class="nav-item">Contact</a></li>
+                    <li class="nav-item">
+                        <a href="#">
+                            <img src="images/icon/Icon - Mail.png" alt="Mail icon" class="mail-icon">
+                        </a>
                     </li>
                 </ul>
-                <a href="#" class="nav-item">
-                    <img src="images/icon/mail-icon.png" alt="Mail icon" class="mail-icon">
-                </a>
             </div>
         </nav>
     </div>
```

Comment:
- For the `headline` section, the background image is not taking the full-width space. (I am using 1440px screen size). Kindly fix this issue.

Figma
<img width="893" alt="Screenshot 2022-12-10 at 10 21 32 PM" src="https://user-images.githubusercontent.com/24832846/206866946-b0addea2-394b-41e9-9fa7-00f708f2935a.png">

Your Design
<img width="1440" alt="Screenshot 2022-12-10 at 10 38 09 PM" src="https://user-images.githubusercontent.com/24832846/206866940-14e6c61b-fe3b-4734-ba22-5f0549f45c45.png">

Reply:
I have fixed background to right with full width.